tests/integrations/test_messagingqueues.py

```diff
@@ -0,0 +1,90 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023 Datadog, Inc.
+
+from utils import weblog, interfaces, scenarios
+from utils.tools import logger
+
+
+@scenarios.integrations
+class Test_MQKafka:
+    def setup_apm_kafka_default_context_propagation(self):
+        self.r = weblog.get("/dsm?integration=kafka")
+
+    def setup_dsm_kafka_default_context_propagation(self):
+        self.r = weblog.get("/dsm?integration=kafka")
+
+    def test_dsm_kafka_default_context_propagation(self):
+        for data in interfaces.agent.get_dsm_data():
+            for stats_bucket in data["request"]["content"].get("Stats", {}):
+
+                producer_stats_point = None
+                consumer_stats_point = None
+
+                for stats_point in stats_bucket.get("Stats", {}):
+                    if stats_point["EdgeTags"][0] == "direction:in":
+                        consumer_stats_point = stats_point
+                    elif stats_point["EdgeTags"][0] == "direction:out":
+                        producer_stats_point = stats_point
+
+                # consumers are a direct child of a producer span
+                assert consumer_stats_point["ParentHash"] == producer_stats_point["Hash"]
+
+    # kafka in dd-trace-py does not propagate span context for tracing purposes
+    def test_apm_kafka_default_context_propagation(self):
```

Comment:
We want to be able to report red/green for support of context propagation, so we don't want to assert *current* behavior, we want to assert *desired* behavior, and then we can x-fail (or not) based on some list of exceptions.

We want to create a support table with the following rows:

Language, System, Library, Propagation Method, Produce Supported?, Consume Supported?

Reply:
Can this be resolved now that the tests for kafka produce/consume span behavior is in https://github.com/DataDog/system-tests/pull/1783 ?